lib/dicon/secure_shell.ex

```diff
@@ -156,6 +156,34 @@ defmodule Dicon.SecureShell do
     |> IO.write
   end
 
+  def tail(%__MODULE__{conn: conn}, patterns, line_transformer, device) do
+    result =
+      with {:ok, channel} <- :ssh_connection.session_channel(conn, @timeout),
+           tail_command = String.to_charlist("tail --follow=name " <> Enum.join(patterns, " ")),
```

Comment:
Sorry, commented on the wrong comment. So, `--retry` and explicit `--follow=name` are documented on `tail` on Linux, but they appear to be not documented on `tail` on macOS. macOS however does document `-F`:

> The `-F` option implies the `-f` option, but tail will also check to see if the file being followed has been renamed or rotated. The file is closed and reopened when tail detects that the filename being read from has a new inode number.

I think it would make sense to have `tail -F` instead, since on Linux `-F` is documented to be equivalent to `--follow=name --retry`. Wdyt?